zikyo-core/shared/src/main/scala/zikyo/package.scala

```diff
@@ -236,6 +236,12 @@ package object zikyo:
         def as[A1, S1](value: => A1 < S1): A1 < (S & S1) =
             effect.map(_ => value)
 
+        def debug: A < (S & IOs) =
+            effect.tap(value => Console.default.println(value.toString))
+
+        def debug(prefix: => String): A < (S & IOs) =
+            effect.tap(value => Console.default.println(s"$prefix: $value"))
```

Comment:
Should we use `Consoles` here instead of the default? Since it's for debugging I was less inclined to include an additional effect to run.

Reply:
I agree that for debug we should use default